csrc/quantization/cutlass_w8a8/c3x/scaled_mm.cuh

```diff
@@ -88,4 +88,68 @@ struct cutlass_3x_gemm {
   struct GemmKernel : public KernelType {};
 };
 
+template <typename ElementAB_, typename ElementD_,
+          template <typename, typename, typename> typename Epilogue_,
+          typename TileShape, typename ClusterShape, typename KernelSchedule,
+          typename EpilogueSchedule>
+struct cutlass_3x_gemm_sm100 {
+  using ElementAB = ElementAB_;
+  using LayoutA = cutlass::layout::RowMajor;
+  static constexpr int AlignmentA =
+      128 / cutlass::sizeof_bits<ElementAB_>::value;
+
+  using LayoutB = cutlass::layout::ColumnMajor;
+  static constexpr int AlignmentB =
+      128 / cutlass::sizeof_bits<ElementAB_>::value;
+
+  using ElementD = ElementD_;
+  using LayoutD = cutlass::layout::ColumnMajor;
+  static constexpr int AlignmentD =
+      128 / cutlass::sizeof_bits<ElementD_>::value;
```

Comment:
We've been setting AlignmentD to 4 to reduce the alignment requirement of these kernels. Can this be 4 instead of 8? Also, do you know what the performance considerations to this are?

Reply:
Generally 128-bit alignment (i.e. 8 for 16-bit data types) is required for best TMA perf, I think 4 might work but perf will suffer and It's the same between Hopper and Blackwell